frontend/src/component/Activity/Preview.jsx

```diff
@@ -48,7 +48,12 @@ class Preview extends React.Component {
   handleUnavailablePreview = () => {
     this.setState({
       previewLoading: false,
-      previewComponent: null,
+      previewComponent: (
```

Comment:
is there a reason for setting it in the state?

I think you can declare your PreviewComponent in a const in the same file

things that aren't meant to change in time doesn't need to be in the state

Reply:
This was a prior request: https://github.com/tracim/tracim/pull/3843#discussion_r526724784
What should I do?

Reply:
note: this is not constant

Reply:
After discussion, we decided to store the html code of the preview instead of a react element and got rid of the  the previewComponent field as a result